### PR TITLE
Fixed issues

### PR DESCRIPTION
**Pull request by logan-mo**
1) The first issue was the older HTTP request to the server returned an error code. After back-and-forth emails with the NASA Power team, they helped me make new HTTTP requests. Essentially, if we reduce the number of features per request, it works. So I modified the code to make two requests, each for half of the features, and then merge them both together.

2) The code would fail at the very last step because there was no folder called dataset. Now Setup.py does that for you.

### CODE REVIEW

File: config/locations_pruned.yml [hunk 1]
```diff
@@ -34,57 +29,22 @@ target_locations:
...
-    power: 1000
-  Chennai Metropolitan Area:
-    location:
-      - 13.083667
-      - 80.268667
   Adani Green Energy Tamilnadu Limited:
     location:
       - 9.332771
       - 78.388731
-  Solar power plant Koppal:
```
Comment (markub3327) [inline config/locations_pruned.yml]: Why you deleted any location from `locations_pruned.yml`? All of these were used in my paper: https://www.mdpi.com/2076-3417/12/17/8852

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import os
+os.system("julia install_packages.jl")
+if not os.path.exists('dataset'):
```
Comment (markub3327) [inline setup.py]: This is a pure Julia project. Please don't combine it with Python.

Creating not existing path can be done in Jula too.

**Review by markub3327 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (logan-mo) [reply setup.py]: Done.  Also, the changes to the locations_pruned.yml were not made by me, but by the code itself.

**Comment by logan-mo** [comment]
Changes done.

File: setup.py (hunk 2 above)
Comment (markub3327) [reply setup.py]: What line of code made these changes?

Comment (markub3327) [reply setup.py]: @logan-mo What line of code made these changes?

Comment (logan-mo) [reply setup.py]: @markub3327 No idea. I don't know Julia at all. But I do know If I run the code, the values in the locations_pruned are modified.

Comment (logan-mo) [reply setup.py]: image
If I had to guess, this part that writes to the file probably does something.

Comment (markub3327) [reply setup.py]: ok.

**Pull request merged**